modules/webapp/src/main/elm/Messages/Comp/UserTable.elm

```diff
@@ -40,5 +40,5 @@ de =
     , email = "E-Mail"
     , logins = "Anmeldungen"
     , lastLogin = "Letzte Anmeldung"
-    , formatDateTime = DF.formatDateTimeLong Messages.UiLanguage.English
+    , formatDateTime = DF.formatDateTimeLong Messages.UiLanguage.German
```

Comment:
Not sure if this was intentionally left english

Reply:
No, was a mistake. Good catch!